templates/taxi/driver_list.html

```diff
@@ -1,8 +1,8 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Driver List
-    </h1>
+    <h1>Driver List</h1>
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary">Create Driver</a>
```

Comment:
Consider adding a 'Cancel' button that leads to the previous page the user was on, as per the checklist requirement.